### DISCUSSION TITLE
Not interested in crosshairs but...

### DISCUSSION THREAD

**Discussion by WOLFGE1ST** [lacymorrow/crossover#338] [General]
I am interested in this app or one like it for another game (Ultima Online). Basically I just want to create an image to stick in each of the 4 corners of my game window, along with some markers to indicate range. Is this feasible through this app or should I look elsewhere?

**Comment by lacymorrow** [comment]
That would be feasible with this app. You would create four duplicate windows, and set a custom image.

The new version (still in development) allows for a different image in each window.
